docs/troubleshoot/verifier/notation.md

```diff
@@ -0,0 +1,106 @@
+      "referenceDigest": "sha256:57be2c1c3d9c23ef7c964bba05c7aa23b525732e9c9af9652654ccc3f4babb0e",
+      "artifactType": "application/vnd.cncf.notary.signature",
+      "verifierReports": [
+        {
+          "isSuccess": false,
+          "message": "Original Error: (Original Error: (artifact \"ghcr.io/deislabs/ratify/notary-image@sha256:8e3d01113285a0e4aa574da8eb9c0f112a1eb979d72f73399d7175ba3cdb1c1b\" has no applicable trust policy. Trust policy applicability for a given artifact is determined by registryScopes. To create a trust policy, see: https://notaryproject.dev/docs/quickstart/#create-a-trust-policy), Error: verify signature failure, Code: VERIFY_PLUGIN_FAILURE, Plugin Name: notation, Component Type: verifier, Documentation: https://github.com/notaryproject/notaryproject/tree/main/specs, Detail: failed to verify signature of digest), Error: verify reference failure, Code: VERIFY_PLUGIN_FAILURE, Plugin Name: notation, Component Type: verifier",
+          "name": "notation",
+          "extensions": null
+        }
+      ],
+      "nestedReports": []
+    }
+  ]
+```
+
+Users can investigate the root cause of the Notation verifier by checking the `message` field of each failed `verifierReport`. The error message could be a nested error, the Notation error is the most inner error, e.g. `artifact \"ghcr.io/deislabs/ratify/notary-image@sha256:8e3d01113285a0e4aa574da8eb9c0f112a1eb979d72f73399d7175ba3cdb1c1b\" has no applicable trust policy. Trust policy applicability for a given artifact is determined by registryScopes. To create a trust policy, see: https://notaryproject.dev/docs/quickstart/#create-a-trust-policy` in the above example.
+
+Since the other levels of the error are always the same, thid TSG would focus on different errors returned by Notation verifier.
+
+## Scenario 1
+```
+artifact URI [uri] could not be parsed, make sure it is the fully qualified OCI artifact URI without the scheme/protocol. e.g domain.com:80/my/repository@sha256:digest
+```
+
+### Cause and Solution
+The provided reference URI doesn't contain character `@` which is required by Notation verifier. Please check the reference URI and make sure it is in the correct format. 
+
+Check [documentation](https://github.com/notaryproject/specifications/blob/main/specs/trust-store-trust-policy.md#selecting-a-trust-policy-based-on-artifact-uri) for more details.
+
+## Scenario 2
+```
+registry scope [scope] is not valid, make sure it is a fully qualified repository without the scheme, protocol or tag. For example domain.com/my/repository or a local scope like local/myOCILayout
+```
+Or
+```
+registry scope [scope] with wild card(s) is not valid, make sure it is a fully qualified repository without the scheme, protocol or tag. For example domain.com/my/repository or a local scope like local/myOCILayout
+```
+
+### Cause and Solution
+The error is self-explanatory.
```

Comment:
Suggestion to remove "error is self-explanatory" , given customer is reading TSG, it suggest they were unable to resolve the issue.